packages/libs/react-ui/src/components/Form/Checkbox/Checkbox.css.ts

```diff
@@ -0,0 +1,111 @@
+import { style } from '@vanilla-extract/css';
+import { atoms, token, uiBaseRegular } from '../../../styles';
+
+export const labelClass = style([
+  atoms({
+    display: 'flex',
+    color: 'text.base.default',
+  }),
+  {
+    alignItems: 'flex-start',
+    lineHeight: token('size.n4'),
+    cursor: 'pointer',
```

Comment:
can alignItems and cursor be in atoms?

Reply:
Just discussed to remove atoms from the react-ui library. So this will change but  I'll be moving the styles out the atoms function.